### PR TITLE
Added Certificate Authorization Authority records and corrected a few…

### PR DESCRIPTION
**Pull request by maartenvhb**
… typos in my name

### CODE REVIEW

File: src/theory/PKIs.tex [hunk 1]
```diff
@@ -150,15 +150,69 @@ \subsection{Hardening PKI}
...
+\subsubsection{Configuration of CAA records}
+\label{sec:pki:caarecords:configuration}
+ 
+BIND supports CAA records as of version 9.9.6.
+ 
+A CAA record can be configured by adding it to the zone file:
+
+\begin{lstlisting}
+$ORIGIN example.com
+       CAA 0 issue "ca1.org"
```
Comment (sebix) [inline src/theory/PKIs.tex]: I'd prefer a reserved domain as defined in [RFC 2606](https://tools.ietf.org/html/rfc2606).

File: src/theory/PKIs.tex [hunk 2]
```diff
@@ -150,15 +150,69 @@ \subsection{Hardening PKI}
...
+\end{lstlisting}
+ 
+If your organization uses multiple CA’s, you can configure multiple records:
+
+\begin{lstlisting} 
+      CAA 0 issue "ca1.org"
+      CAA 0 issue "ca2.org"
+\end{lstlisting}
+ 
+“ca1.org” and “ca2.org” are unique identifiers for the CA you plan on using. These strings can be obtained from your Certificate Authority, and typically are its top level domain. An example is “letsencryptorg” for the Let’s Encrypt CA operated by the Internet Security Research Group.
```
Comment (sebix) [inline src/theory/PKIs.tex]: Do you mean `“letsencrypt.org”` instead of `“letsencryptorg”`?

**Comment by maartenvhb** [comment]
Thanks @sebix, appreciate the comments. I made these recommended changes.

**Comment by azet** [comment]
LGTM.

**Pull request merged**